.github/workflows/ci.yml

```diff
@@ -0,0 +1,24 @@
+name: CI
```

Comment:
there is an open PR adding CI here: https://github.com/kernelci/logspec/pull/9/commits/a0741baabb9374ecdf82c1ee168f0fe733a41242